network/websocket_dataframe.ksy

```diff
@@ -0,0 +1,68 @@
+meta:
+  id: websocket_dataframe
+  title: WebSocket
+  xref:
+    rfc: 6455
+  endian: be
+  license: APACHE-2.0
+doc: |
+  The WebSocket protocol establishes a two-way communication channel via TCP.
+  Messages are made up of one or more dataframes, and are delineated by
+  frames with the `fin` bit set.
+seq:
+  - id: fin
+    type: b1
+  - id: rsv1
```

Comment:
Can be merged. And probably should.

Reply:
Also, as per [style guide](http://doc.kaitai.io/ksy_style_guide.html), it's better to stick to `reserved` name:

> For reserved fields which are known to be unused, use `reserved` name (or `reserved1`, `reserved2`, etc, if there are many of them)

Reply:
What do you mean by merging these? Moving them to a separate types block?

Reply:
```yaml
- id: reserved
  type: b3
```